Library/Homebrew/test/build_environment_spec.rb

```diff
@@ -0,0 +1,70 @@
+require "build_environment"
+
+RSpec::Matchers.define :use_userpaths do
+  match do |actual|
+    actual.userpaths?
+  end
+end
+
+describe BuildEnvironment do
+  let(:env) { described_class.new }
+
+  describe "#<<" do
+    it "returns itself" do
+      expect(env << :foo).to eq env
```

Comment:
You've changed the behaviour of these tests. The Minitest equivalents used `assert_same`, which checks instance equality using `#equal?`. The equivalent construction in RSpec is `be` (instead of `eq`).